sdk/worker.go

```diff
@@ -95,6 +95,7 @@ type Model struct {
 	Template         *map[string]string `json:"template"  db:"template" cli:"-"`
 	RunScript        string             `json:"run_script"  db:"run_script" cli:"-"`
 	Disabled         bool               `json:"disabled"  db:"disabled" cli:"disabled"`
+	Restricted       bool               `json:"restricted"  db:"restricted" cli:"restricted"`
```

Comment:
don't you have to regenerated the easyjson stuff ?

Reply:
done